ext/search_kit/tests/phpunit/api/v4/SearchDisplay/SearchDisplayTest.php

```diff
@@ -50,4 +51,44 @@ public function testGetDefaultNoEntity() {
     $this->assertNull($display['saved_search_id']);
   }
 
+  public function testAutoFormatName() {
```

Comment:
non blocking - but it's better to specify return type hint when adding new functions

Reply:
It feels silly adding `:void`. I mean, a function doesn't have to *do* anything to return void. You don't have to put `return void;` at the end of the function before the closing bracket. It's just the default behavior.

Reply:
@colemanw yeah - it is  where phpunit has gone though - and I think in general seen to be the way php is going

Reply:
The default return type is actually "mixed" though, right? It would be better if it was void. But history.